parsons/bluelink/person.py

```diff
@@ -0,0 +1,242 @@
+import logging
+import json
+
+logger = logging.getLogger(__name__)
+
+
+class Person(object):
+    """
+    Instantiate Person Class.
+    Used for to upserting via Bluelink connector.
+    See: https://bluelinkdata.github.io/docs/BluelinkApiGuide#person-object
+
+    `Args:`
+        identifiers: list[Identifier]
+            A list of Identifier objects.
+            A Person must have at least 1 identifier
+        given_name: str
+            First name / given name
+        family_name: str
+            Last name / family name
+        emails: list[Email]
+            A list of Email objects representing email addresses.
+        phones: list[Phone]
+            A list of Phone objects representing phone numbers.
+        addresses: list[Address]
+            A list of Address objects representing postal addresses
+        tags: list[Tag]
+            Simple tags that apply to the person, eg DONOR.
+        employer: str
+            Name of the persons employer
+        employer_address: Address
+            Address of the persons employer
+        occupation: str
+            Occupation
+        birthdate: str
+            ISO 8601 formatted birth date: YYYY-MM-DD
+        scores: list[Score]
+            List of Score objects. Scores are numeric scores, ie partisanship model.
+        details: dict
+            additional custom data. must be json serializable
+    """
+    def __init__(self, identifiers, given_name=None, family_name=None, phones=None, emails=None,
+                 addresses=None, tags=None, flags=None, employer=None, employer_address=None,
+                 occupation=None, scores=None, birthdate=None, details=None):
+
+        if not identifiers:
+            raise Exception("Person requires list of Identifiers with at least 1 Identifier")
+
+        self.identifiers = identifiers
+        self.addresses = addresses
+        self.emails = emails
+        self.phones = phones
+        self.tags = tags
+        self.flags = flags
+        self.scores = scores
+
+        self.given_name = given_name
+        self.family_name = family_name
+
+        self.employer = employer
+        self.employer_address = employer_address
+        self.occupation = occupation
+        self.birthdate = birthdate  # TODO: verify is ISO
```

Comment:
Probably makes sense to remove these TODOs before making the final PR?

Reply:
Okay, I removed them. 

There aren't many TODOs in the code base, just a few. Personally, I think it would have been okay to leave em there, but I also understand why we might not have wanted to.